test/integration/integration_test.exs

```diff
@@ -49,13 +49,21 @@ defmodule KafkaEx.Integration.Test do
     assert consumer_group_update_interval == 30000
   end
 
-  test "create_worker provides a default consumer_group of 'kafka_ex'" do
```

Comment:
Actually, I could argue that magically doing anything for the user doesn't help them understand Kafka, which is already fairly murky.  But I think it's within your prerogative as a maintainer to want to continue to provide a default.  

I do, however, think the kinks need to be worked out.  If you want the default to be "kafka_ex", then that should be the setting in the config.exs file.  Disabling consumer groups by setting the name of the consumer group to false is indirect and I think that's evidenced by the complication we see in server.ex, where the consumer group's name is used and it's entirely possible to have autocommit set to true in conditions when consumer groups should theoretically be unsupported.